### DISCUSSION TITLE
How do I attach a window to a tab in an existing OS Window

### DISCUSSION THREAD

**Discussion by gzagatti** [kovidgoyal/kitty#5889] [General]
I have mapped

```
map kitty_mod+d detach_window ask
```

However, the prompt does not include the option to attach to tabs in other OS Windows. How could I achieve this option?

**Comment by kovidgoyal** [comment]
Yes, it does. Remember that it wil lwork only wor new OS windows in the
*same kitty process*. if you start multiple kitty processes, use
--single-instance.

**Comment by gzagatti** [comment]
Thanks for the solution and fast response time! The solution is exactly what I was looking for.
